canvas_modules/common-canvas/__tests__/common-properties/controls/expression-test.js

```diff
@@ -491,8 +491,9 @@ describe("expression-builder renders correctly", () => {
 			expect(expectedReturnType).to.eql(actualReturnType);
 		});
 	});
-	// TODO cannot find valuesTable
-	it.skip("expression builder displays table header descriptions in info icon", async() => {
+
+	// TODO Works in UI, not updating in test Values table field header returns "Value" instead of "CustomValue".
+	it.skip("expression builder displays table header descriptions in info icon", () => {
```

Comment:
Since this works OK in the UI, can you add a Cypress test for this and remove this test from Jest.